src/packages/@ncigdc/containers/FileAggregations.js

```diff
@@ -147,7 +147,7 @@ export type TProps = {
     doc_type: String,
     field: String,
     full: String,
-    type: 'id' | 'string' | 'long',
+    type: 'id' | 'string' | 'long' | 'float' | 'double',
```

Comment:
Interesting that `float` wasn't in there before

Reply:
These are typings for flow. They don't actually change how the js executes.  But since these typing existed, I updated them to be more accurate.  fwiw, flow lost to typescript in the "typed javascript" war.